components/common/Tabs/tabs.tsx

```diff
@@ -28,45 +28,70 @@ interface TabsProps extends VariantProps<typeof tabsCva> {
   children: React.ReactElement[]
 }
 
-const Tabs = ({ size, tabs, children }: TabsProps) => (
-  <Tab.Group>
-    <Tab.List className="no-scrollbar flex overflow-x-auto bg-white">
-      {tabs.map(name => (
-        <Tab className={tabsCva({ size })} key={name}>
-          {({ selected }) => (
-            <>
-              {selected && (
-                <motion.div
-                  className="absolute bottom-0 left-0 h-[2px] w-full bg-primaryShade-50 shadow-[0_-1px_6px_rgba(242,24,61,0.24)]"
-                  layoutId="tabs-selected-underline"
-                />
-              )}
-              <span
-                className={clsx('relative', selected && 'text-transparent')}
-              >
-                {name}
+const Tabs = ({ size, tabs, children }: TabsProps) => {
+  const firstTab = tabs[0]
+  const lastTab = tabs[tabs.length - 1]
+
+  return (
+    <Tab.Group>
+      <Tab.List className="no-scrollbar flex overflow-x-auto bg-white">
+        {tabs.map(name => (
+          <Tab
+            onClick={(e: any) => {
+              setTimeout(() => {
+                if (lastTab === name) {
+                  e.target.scrollIntoView({
+                    block: 'end',
+                    inline: 'nearest',
+                    behavior: 'smooth',
+                  })
+                } else if (firstTab === name) {
+                  e.target.scrollIntoView({
+                    block: 'start',
+                    inline: 'nearest',
+                    behavior: 'smooth',
+                  })
+                }
+              }, 100)
+            }}
```

Comment:
`block` is for vertical alignment so all this doesn't actually do anything, it still scrolls to the end of text
`inline` is for horizontal alignment

solution below works like you desired
```suggestion
            onClick={(e: any) => {
              setTimeout(() => {
                e.target.scrollIntoView({
                  inline: tabs[0] === name ? 'start' : 'end',
                  behavior: 'smooth',
                })
              }, 100)
            }}
```

Reply:
😮